predictsignauxfaibles/data.py

```diff
@@ -38,6 +38,7 @@ def __init__(  # pylint: disable=too-many-arguments
         sirets: List = None,
         sirens: List = None,
         outcome: bool = None,
+        **categorical_filters,
```

Comment:
Let's keep `**categorical_filters`

Reply:
There's pros and cons in both cases. If we ever add other functionalities that imply new keyword arguments outside categorical filters, I would be in favour of renaming **categorical_filters to **kwargs. This can be done in a future commit.